website/docs/components/carousel/carousel-a11y.md

```diff
@@ -10,32 +10,50 @@ tabs: Design('carousel'), A11y('carousel-a11y'), API('carousel-api'), Example('c
 
 Table: Keyboard support
 
-| Key                             | Function                                              |
-| ------------------------------- | ----------------------------------------------------- |
-| `Tab`                           | Moves focus to the carousel.                          |
-| `Shift + Tab`                   | Moves focus to the previous focusable element.        |
-| `Left Arrow` , `Right Arrow`    | Display next or previous slide in the carousel.       |
+| Key                         | Function                                                                                                                                                   |
+| --------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `Tab`                       | Moves focus through interactive elements in the carousel.                                                                                                  |
+| `Shift + Tab`               | Moves focus to the previous focusable element.                                                                                                             |
```

Comment:
I'd merge `shift+tab` with `tab`, leaving just the 1st description. It will be more concise, and eliminate possible confusion due to the difference between the 1st and 2nd row descriptions.